tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py27, py3
+
+[testenv]
+whitelist_externals= /bin/bash
+                     /usr/bin/wget
+                     /usr/bin/java
+deps= git+https://github.com/uc-cdis/doiclient.git
```

Comment:
Can we use
```
    -rrequirements.txt
    -rtest-requirements.txt
```
here to install the deps like in the other tox PRs? Then I think we'll be able to drop the `pip install` and `setup.py install` commands.